src/api/getVaultIdFromServer.js

```diff
@@ -0,0 +1,8 @@
+const validateAssertions = require('../lib/validators/validateAssertions')
+
+module.exports = (dvf, token, nonce, signature) => dvf.postAuthenticated(
+  '/v1/trading/r/getVaultId',
```

Comment:
Looks like the intention is to move away from spare vault mechanism. Should be fine as lon gas we add getVaultId to pub api.

Reply:
it's already implemented: https://github.com/ethfinex/launch-deversifi/commit/2b6d9cf25579e8a29375240fc4b66bccdad23e7e